core/Cargo.toml

```diff
@@ -325,3 +325,7 @@ zksync_contract_verification_server = { version = "26.2.1-non-semver-compat", pa
 zksync_node_api_server = { version = "26.2.1-non-semver-compat", path = "node/api_server" }
 zksync_base_token_adjuster = { version = "26.2.1-non-semver-compat", path = "node/base_token_adjuster" }
 zksync_logs_bloom_backfill = { version = "26.2.1-non-semver-compat", path = "node/logs_bloom_backfill" }
+
+[patch.crates-io]
+# Needed to include https://github.com/eigerco/lumina/commit/f59466a25e5392d01a8e15a59b117766e3aa2cc4
+celestia-types = {git="https://github.com/eigerco/lumina.git"}
```

Comment:
```suggestion
```

0.10.1 is up now, just need to rm the patch here & do `cargo update -p celestia-types`